kentik_api_library/tests/e2e/README.md

```diff
@@ -1,3 +1,20 @@
-# e2e tests
+# End-to-End (e2e) tests
 
-E2E tests connect to production Kentik server and require valid credentials passed as KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN env variables.
\ No newline at end of file
+The e2e test suite verifies:
+- CRUD operations for all supported test types
+- listing of all tests
+- listing of all agents
+
+It interacts with production Kentik API and requires account with sufficient synthetic test
+credits. User credentials myst be passed in KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN env variables.
```

Comment:
myst -> must

Reply:
Oops, I've missed this typo and your comment before merging. Can you, please, fix it in [your cloudexports PR](https://github.com/kentik/community_sdk_python/pull/122)?